### PR TITLE
Added German translation

### PR DESCRIPTION
**Pull request by betaboot**
Everything is translated to German, including the latest changes.

### CODE REVIEW

**Comment by betaboot** [comment]
uh oh, i guess i shouldn't have merged your master branch into this one. can that be fixed?

**Comment by VoidingWarranties** [comment]
Thanks for the PR!

No worries, typically we use `git rebase master` instead of merging in order to keep a clean commit history. There are pros and cons of both and some teams prefer rebasing while others prefer merging. You can fix this  like so:

1. `git rebase -i HEAD~3`
2. Delete the 5 commits from the merge in between your first commit and your last commit (5256123, 8bdb194, 1641338, 2642635, 3276fb8)
3. `git rebase master`
4. `git push origin master`

File: i18n/de.yaml [hunk 1]
```diff
@@ -0,0 +1,636 @@
...
+  translation: "Français"
+- id: ru
+  translation: "Русский"
+- id: zh-hans
+  translation: "中文"
+- id: zh-hant
+  translation: "繁體中文"
+- id: de
+  translation: "Deutsch"
+
```
Comment (VoidingWarranties) [inline i18n/de.yaml]: Could you add the translation for `de` to `en.yaml`, and then delete this line and above in this file?

It's an oddity with the way the translation library works with Hugo. Not entirely sure that it's not a bug to be honest, but nonetheless the workaround is easy.

**Comment by betaboot** [comment]
Thanks for the help! I did that and hope it is correct now.
I'm happy to do some stuff with Git, always wanted to learn it  :)

**Pull request merged**